ci/launch_cluster.sh

```diff
@@ -27,6 +27,12 @@ fi
 CHANNEL="$1"
 VARIANT="$2"
 DEPLOYMENT_NAME="$3"
+EXPIRATION="2hours"
```

Comment:
what is the default expiration in dcos_launch if we don't specify anything? Is it 2h?